docs/epytext_demo/demo_epytext_module.py

```diff
@@ -8,6 +8,17 @@
 from somelib import SomeInterface
 import zope.interface
 import zope.schema
+from typing import Final
+
+LANG = 'Fr'
+"""
+This is a constant.
+"""
+
+lang: Final = ['Fr', 'En']
```

Comment:
Maybe something like this for complete typing ?

```suggestion
lang: Final[Sequence[str]] = ['Fr', 'En']
```

Reply:
That's interesting stuff, not even sure `typing. Final[something]` will actually work!

Let me check...

Reply:
Whether it already works or not, it would be good to have a unit test case for it.

Reply:
Spoiler: it's not working @mthuurne